templates/digital-asset-template/move/sources/launchpad.move

```diff
@@ -1,4 +1,4 @@
-module launchpad_addr::nft_launchpad {
+module launchpad_addr::launchpad {
```

Comment:
why this change?

Reply:
the package name is digital_asset_launchpad, want to keep it consistent, though we still call it nft inside the contract.. but i tried to not call it nft in module name.